src/main/java/core/basesyntax/sevice/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.sevice.impl;

```

Comment:
The package name should be `service.impl` instead of `sevice.impl` to follow the standard naming convention and project structure mentioned in the checklist.